{"Timestamp":"2025-03-10T13:38:03.1692631+01:00","Level":"Error","MessageTemplate":"An error occurred using the connection to database '{database}' on server '{server}'.","Properties":{"database":"","server":"localhost","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection"}}
{"Timestamp":"2025-03-10T15:26:04.0226488+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"aa270a9b93ca3005f8e58a9d57d7d75c","SpanId":"4c70e1fc7e457ada","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVRTUI5J8I:00000001","RequestPath":"/","ConnectionId":"0HNAVRTUI5J8I"}}
{"Timestamp":"2025-03-10T15:28:38.9097610+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"4297c0b611056ab832314dedd9dcaa88","SpanId":"0240a2ff25c5d332","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVRVCNOJ3M:00000001","RequestPath":"/","ConnectionId":"0HNAVRVCNOJ3M"}}
{"Timestamp":"2025-03-10T15:29:15.1552529+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"d07817bfa2d63f41cc75393130cc8aac","SpanId":"14c9c183e97025f0","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVRVCNOJ3M:00000009","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVRVCNOJ3M"}}
{"Timestamp":"2025-03-10T15:30:13.6425419+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"0867cb666f3497758c05d7c25ce354c5","SpanId":"751adf31ac978c24","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVS08VB8E3:00000001","RequestPath":"/","ConnectionId":"0HNAVS08VB8E3"}}
{"Timestamp":"2025-03-10T15:30:19.6688520+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"06cdbbb05abde1f5d1e616986eaad814","SpanId":"500d0eec67efebc9","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVS08VB8E3:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVS08VB8E3"}}
{"Timestamp":"2025-03-10T15:41:02.0853814+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"4f8e8c1cec7b056b0bd020273d02d987","SpanId":"662f44d3c227ce9e","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVS6A70TJ0:00000001","RequestPath":"/","ConnectionId":"0HNAVS6A70TJ0"}}
{"Timestamp":"2025-03-10T15:41:07.0880380+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"b3fa5744647a43ce06b7a467d4f72fd1","SpanId":"01c64c03a017133f","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVS6A70TJ0:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVS6A70TJ0"}}
{"Timestamp":"2025-03-10T15:42:15.7138247+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"2ca7f38c57b4ecc3506f7618122628be","SpanId":"136283f0a21c6342","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVS6A70TJ3:00000003","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVS6A70TJ3"}}
{"Timestamp":"2025-03-10T15:54:33.6474816+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"3509e968a4504286c680b51ff88ef322","SpanId":"bdcce68a65a62951","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVS6A70TJ6:00000003","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVS6A70TJ6"}}
{"Timestamp":"2025-03-10T16:13:33.6516692+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"210ee5dbdf01ff0944c802c03907cdbf","SpanId":"39718f1700d7c539","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVSOFQLENI:00000001","RequestPath":"/","ConnectionId":"0HNAVSOFQLENI"}}
{"Timestamp":"2025-03-10T16:13:41.7046023+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"f91918f9cada18ab19f9d0372664f08c","SpanId":"c51d8a4bbd0d82e1","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSOFQLENI:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSOFQLENI"}}
{"Timestamp":"2025-03-10T16:14:33.4329016+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"9f97c5ecef2b224c1c818afdd2ce7d12","SpanId":"18c6d6d0faf66db4","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVSP1KQVF3:00000001","RequestPath":"/","ConnectionId":"0HNAVSP1KQVF3"}}
{"Timestamp":"2025-03-10T16:14:39.2097267+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"6066c3fb2aa48309753fd0b6457e9c3d","SpanId":"01fed3ff1b014c57","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSP1KQVF3:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSP1KQVF3"}}
{"Timestamp":"2025-03-10T16:16:42.0011704+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"0d0e6268a81830e528ea153717d8c133","SpanId":"ce0a829f0f8c338a","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVSQ7V5EOC:00000001","RequestPath":"/","ConnectionId":"0HNAVSQ7V5EOC"}}
{"Timestamp":"2025-03-10T16:16:48.7323744+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"b6b2538f3c89dc7b0821c879b3af6eed","SpanId":"a998cd9ddfae962d","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSQ7V5EOC:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSQ7V5EOC"}}
{"Timestamp":"2025-03-10T16:19:11.5347921+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"45896f83bb41a7262266672990d9b572","SpanId":"5ee77e80559fb77c","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSQ7V5EOF:00000001","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSQ7V5EOF"}}
{"Timestamp":"2025-03-10T16:19:16.1647043+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"1a0d3d41fcbb4a488827e0d83c0b995b","SpanId":"2486df62bcf9dedf","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSQ7V5EOF:00000004","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSQ7V5EOF"}}
{"Timestamp":"2025-03-10T16:20:11.0098061+01:00","Level":"Warning","MessageTemplate":"'{CorrelationCookieName}' cookie not found.","TraceId":"cd15963efe459f0ea8e186b19a3c7272","SpanId":"8bc9359c3e8be897","Properties":{"CorrelationCookieName":".AspNetCore.Correlation.2fxwi6E4ULdvY4bepgd8f6lPRA0QD2n1c0DJRMpqNHs","EventId":{"Id":15,"Name":"CorrelationCookieNotFound"},"SourceContext":"Microsoft.AspNetCore.Authentication.Google.GoogleHandler","RequestId":"0HNAVSQ7V5EOF:00000005","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSQ7V5EOF"}}
{"Timestamp":"2025-03-10T16:20:11.0118624+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"cd15963efe459f0ea8e186b19a3c7272","SpanId":"8bc9359c3e8be897","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: Correlation failed.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSQ7V5EOF:00000005","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSQ7V5EOF"}}
{"Timestamp":"2025-03-10T16:23:33.7938021+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"ba38aeb0aa15557c19a95b4092fbfa93","SpanId":"992730d8e8d15e19","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVSU2M0JFT:00000001","RequestPath":"/","ConnectionId":"0HNAVSU2M0JFT"}}
{"Timestamp":"2025-03-10T16:23:42.6990562+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"f86f6fe4a23f131809f6958ede5263a0","SpanId":"be6bec3fd78862a6","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSU2M0JFT:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSU2M0JFT"}}
{"Timestamp":"2025-03-10T16:24:15.7235652+01:00","Level":"Error","MessageTemplate":"Hosting failed to start","Exception":"System.IO.IOException: Failed to bind to address http://127.0.0.1:80: address already in use.\r\n ---> Microsoft.AspNetCore.Connections.AddressInUseException: Only one usage of each socket address (protocol/network address/port) is normally permitted.\r\n ---> System.Net.Sockets.SocketException (10048): Only one usage of each socket address (protocol/network address/port) is normally permitted.\r\n   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, Boolean disconnectOnFailure, String callerName)\r\n   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)\r\n   at System.Net.Sockets.Socket.Bind(EndPoint localEP)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportOptions.CreateDefaultBoundListenSocket(EndPoint endpoint)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()\r\n   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportFactory.BindAsync(EndPoint endpoint, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.TransportManager.BindAsync(EndPoint endPoint, ConnectionDelegate connectionDelegate, EndpointConfig endpointConfig, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.<>c__DisplayClass28_0`1.<<StartAsync>g__OnBind|0>d.MoveNext()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.LocalhostListenOptions.BindAsync(AddressBindContext context, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.AddressesStrategy.BindAsync(AddressBindContext context, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindAsync(ListenOptions[] listenOptions, AddressBindContext context, Func`2 useHttps, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.BindAsync(CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)\r\n   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__15_1(IHostedService service, CancellationToken token)\r\n   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)","Properties":{"EventId":{"Id":11,"Name":"HostedServiceStartupFaulted"},"SourceContext":"Microsoft.Extensions.Hosting.Internal.Host"}}
{"Timestamp":"2025-03-10T16:24:55.7329516+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"91b97c3d388f795c51557ecc0f39f7fd","SpanId":"96d82916dbdde4a2","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSUM5ET4O:00000019","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSUM5ET4O"}}
{"Timestamp":"2025-03-10T16:26:55.1121277+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"be803ca74bc406162fc1affce7afdfa8","SpanId":"63f201fcd8614387","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVSVUM7DO3:00000001","RequestPath":"/","ConnectionId":"0HNAVSVUM7DO3"}}
{"Timestamp":"2025-03-10T16:27:08.1345428+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"2e38eec046f2b205e683a6405f0b6756","SpanId":"b122742ef3316cf7","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVSVUM7DO3:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVSVUM7DO3"}}
{"Timestamp":"2025-03-10T16:30:45.1138147+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"851a9f13c72c7dfefa8fbb7ac445cdde","SpanId":"6da9301d8d1bba20","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVT237RSV0:00000001","RequestPath":"/","ConnectionId":"0HNAVT237RSV0"}}
{"Timestamp":"2025-03-10T16:30:50.9905600+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"22dc39f24a630750b78cd36744867821","SpanId":"d83a1a0e6109a832","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVT237RSV0:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVT237RSV0"}}
{"Timestamp":"2025-03-10T16:31:55.4367911+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"48ae3099a7013fb56b2984a9b1e26eb8","SpanId":"820c5ff8e80546ff","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVT2O6G02E:00000001","RequestPath":"/","ConnectionId":"0HNAVT2O6G02E"}}
{"Timestamp":"2025-03-10T16:31:55.5037467+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"48ae3099a7013fb56b2984a9b1e26eb8","SpanId":"820c5ff8e80546ff","Exception":"System.ArgumentException: The path in 'value' must start with '/'. (Parameter 'value')\r\n   at Microsoft.AspNetCore.Http.PathString..ctor(String value)\r\n   at Microsoft.AspNetCore.Http.PathString.FromUriComponent(String uriComponent)\r\n   at Microsoft.AspNetCore.Http.PathString.op_Implicit(String s)\r\n   at Program.<>c.<<Main>$>b__0_3(GoogleOptions options) in C:\\Users\\Jonas\\RiderProjects\\Emne9\\Emne9_Prosjekt\\Emne9_Prosjekt\\Program.cs:line 63\r\n   at Microsoft.Extensions.Options.ConfigureNamedOptions`1.Configure(String name, TOptions options)\r\n   at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.<>c.<Get>b__10_0(String name, IOptionsFactory`1 factory)\r\n   at Microsoft.Extensions.Options.OptionsCache`1.<>c__DisplayClass3_1`1.<GetOrAdd>b__2()\r\n   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)\r\n   at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)\r\n   at System.Lazy`1.CreateValue()\r\n   at Microsoft.Extensions.Options.OptionsCache`1.GetOrAdd[TArg](String name, Func`3 createOptions, TArg factoryArgument)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.Get(String name)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.InitializeAsync(AuthenticationScheme scheme, HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandlerProvider.GetHandlerAsync(HttpContext context, String authenticationScheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVT2O6G02E:00000001","RequestPath":"/","ConnectionId":"0HNAVT2O6G02E"}}
{"Timestamp":"2025-03-10T16:37:15.7246496+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"f966eda3632938a38389d2a03629210d","SpanId":"aaf9c89a2621e75d","Exception":"System.ArgumentException: The path in 'value' must start with '/'. (Parameter 'value')\r\n   at Microsoft.AspNetCore.Http.PathString..ctor(String value)\r\n   at Microsoft.AspNetCore.Http.PathString.FromUriComponent(String uriComponent)\r\n   at Microsoft.AspNetCore.Http.PathString.op_Implicit(String s)\r\n   at Program.<>c.<<Main>$>b__0_3(GoogleOptions options) in C:\\Users\\Jonas\\RiderProjects\\Emne9\\Emne9_Prosjekt\\Emne9_Prosjekt\\Program.cs:line 63\r\n   at Microsoft.Extensions.Options.ConfigureNamedOptions`1.Configure(String name, TOptions options)\r\n   at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.<>c.<Get>b__10_0(String name, IOptionsFactory`1 factory)\r\n   at Microsoft.Extensions.Options.OptionsCache`1.<>c__DisplayClass3_1`1.<GetOrAdd>b__2()\r\n   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)\r\n--- End of stack trace from previous location ---\r\n   at System.Lazy`1.CreateValue()\r\n   at Microsoft.Extensions.Options.OptionsCache`1.GetOrAdd[TArg](String name, Func`3 createOptions, TArg factoryArgument)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.Get(String name)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.InitializeAsync(AuthenticationScheme scheme, HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandlerProvider.GetHandlerAsync(HttpContext context, String authenticationScheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVT2O6G02F:00000001","RequestPath":"/wpad.dat","ConnectionId":"0HNAVT2O6G02F"}}
{"Timestamp":"2025-03-10T16:38:03.5511969+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"4eea2a51adf6c5711846a4db86d0babf","SpanId":"f2c192eaa0aa473c","Exception":"System.ArgumentException: The path in 'value' must start with '/'. (Parameter 'value')\r\n   at Microsoft.AspNetCore.Http.PathString..ctor(String value)\r\n   at Microsoft.AspNetCore.Http.PathString.FromUriComponent(String uriComponent)\r\n   at Microsoft.AspNetCore.Http.PathString.op_Implicit(String s)\r\n   at Program.<>c.<<Main>$>b__0_3(GoogleOptions options) in C:\\Users\\Jonas\\RiderProjects\\Emne9\\Emne9_Prosjekt\\Emne9_Prosjekt\\Program.cs:line 63\r\n   at Microsoft.Extensions.Options.ConfigureNamedOptions`1.Configure(String name, TOptions options)\r\n   at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.<>c.<Get>b__10_0(String name, IOptionsFactory`1 factory)\r\n   at Microsoft.Extensions.Options.OptionsCache`1.<>c__DisplayClass3_1`1.<GetOrAdd>b__2()\r\n   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)\r\n--- End of stack trace from previous location ---\r\n   at System.Lazy`1.CreateValue()\r\n   at Microsoft.Extensions.Options.OptionsCache`1.GetOrAdd[TArg](String name, Func`3 createOptions, TArg factoryArgument)\r\n   at Microsoft.Extensions.Options.OptionsMonitor`1.Get(String name)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.InitializeAsync(AuthenticationScheme scheme, HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandlerProvider.GetHandlerAsync(HttpContext context, String authenticationScheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVT2O6G02G:00000001","RequestPath":"/wpad.dat","ConnectionId":"0HNAVT2O6G02G"}}
{"Timestamp":"2025-03-10T16:47:20.7468392+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"aa532bfffd12800ebef2c69c324fa82c","SpanId":"92c2701fc3645dc5","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVTBBUR13B:00000001","RequestPath":"/","ConnectionId":"0HNAVTBBUR13B"}}
{"Timestamp":"2025-03-10T16:47:27.3790149+01:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","TraceId":"790074d2986ce09bcd04d79c89a7fbb0","SpanId":"049f51dda28075be","Exception":"Microsoft.AspNetCore.Authentication.AuthenticationFailureException: An error was encountered while handling the remote login.\r\n ---> Microsoft.AspNetCore.Authentication.AuthenticationFailureException: The oauth state was missing or invalid.\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1.HandleRequestAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HNAVTBBUR13B:00000007","RequestPath":"/api/members/GoogleCallBack","ConnectionId":"0HNAVTBBUR13B"}}
{"Timestamp":"2025-03-10T16:52:55.4188902+01:00","Level":"Warning","MessageTemplate":"Failed to determine the https port for redirect.","TraceId":"53097368d4a43fecbf74610cb6e65592","SpanId":"4cf20560f586e58d","Properties":{"EventId":{"Id":3,"Name":"FailedToDeterminePort"},"SourceContext":"Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware","RequestId":"0HNAVTEFLKG25:00000001","RequestPath":"/","ConnectionId":"0HNAVTEFLKG25"}}
